### ISSUE TITLE
[FR] - validation of the protocol param updates in cardano-cli

### ISSUE THREAD

**Issue by mkoura** [IntersectMBO/cardano-node#2816]
*Internal*

**Area**
*Other*   update proposals

**Describe the feature you'd like**
There is a missing validation for protocol parameters updates in different eras. It is possible to submit update proposal for `minUTxOValue` in Alonzo era (the param is always `null` there) without any error message from CLI.

**Comment by dcoutts** [comment]
Agreed. We do need that. There are TODOs in the code to the same effect.

**Comment by Jimbo4350** [comment]
Closing this. If this is still relevant please reopen.

**Comment by mkoura** [comment]
Still relevant.
